translator/trace/jaeger/jaegerproto_to_traces.go

```diff
@@ -103,10 +103,10 @@ func jProcessToInternalResource(process *model.Process, dest pdata.Resource) {
 
 	attrs := dest.Attributes()
 	if serviceName != "" {
-		attrs.InitEmptyWithCapacity(len(tags) + 1)
+		attrs.EnsureCapacity(len(tags) + 1)
 		attrs.UpsertString(conventions.AttributeServiceName, serviceName)
 	} else {
-		attrs.InitEmptyWithCapacity(len(tags))
+		attrs.EnsureCapacity(len(tags))
```

Comment:
And here.

Reply:
Not a problem, the output param is empty.